### PR TITLE
Fix variable number of reviews returned #676

### PR DESCRIPTION
**Pull request by petskratt**
According to issue #676 reviews (and presumably other listings) that require pagination are returning inconsistent results. Implementing cookieJar appears to fix that.

Not tested with other functions.

### CODE REVIEW

**Comment by petskratt** [comment]
All fails appear to be apps that have been removed from store - `air.com.zinkia.playset` (app method), `air.tv.ingames.cubematch.free` and `com.skybornegames.battlepop` (permisssions method).

File: lib/utils/request.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+import { CookieJar } from 'tough-cookie';
 import createDebug from 'debug';
 
+const cookieJar = new CookieJar();
 const debug = createDebug('google-play-scraper');
 
 function doRequest (opts, limit) {
   let req;
+
+  opts.cookieJar = cookieJar;
```
Comment (facundoolano) [inline lib/utils/request.js:12]:
```suggestion
  // cookies are necessary for pagination to work consistently across requests
  opts.cookieJar = cookieJar;
```

**Comment by facundoolano** [comment]
this LGTM. perhaps you can extend the pagination related test to check that the same query returns the same results every time (ideally this would be failing in main and would be fixed by this change)

**Comment by petskratt** [comment]
Good point - added test that utilises built-in paging, fails on main and works with cookies. As several other tests fail due to discontinued apps ... com.facebook.katana will hopefully stay up for some time and considering the number of reviews perhaps the set of 1500 most helpful ones does not change too often.

**Pull request merged**